demos/stable-diffusion/config.json

```
@@ -0,0 +1,33 @@
+{
+    "model_config_list":[
+       {
+          "config":{
+             "name":"unet",
+             "base_path":"/models/unet",
+             "layout": {"latent_model_input":"...:NCHW","t":"...:...","encoder_hidden_states":"...:CHW"},
```

```suggestion
             "layout": {"latent_model_input":"...:NCHW","t":"...","encoder_hidden_states":"...:CHW"},
```
If you want to reinterpret the layout and not change it, do not use `:`